website/source/docs/platform/k8s/helm.html.md

```diff
@@ -99,19 +101,44 @@ and consider if they're appropriate for your deployment.
        GOOGLE_CREDENTIALS: /vault/userconfig/myproject/myproject-creds.json
     ```
 
+  * `extraSecretEnvironmentVars` (`string: null`) - The extra environment variables populated from a secret to be applied to the Vault server.  This should be a multi-line key/value string.
+
+      - `envName` (`string: required`) -
+      Name of the environment variable to be populated in the Vault container.
+      
+      - `secretName` (`string: required`) -
+      Name of Kubernetes secret used to populate the environment variable defined by `envName`.
+
+      - `secretKey` (`string: required`) -
+      Name of the key where the requested secret value is located in the Kubernetes secret.
+
+    ```yaml
+    # Extra Environment Variables populated from a secret.
+     extraSecretEnvironmentVars:
+      - envName: AWS_SECRET_ACCESS_KEY
+        secretName: vault
+        secretKey: AWS_SECRET_ACCESS_KEY
+    ```
+
   * `extraVolumes` (`array: []`) - A list of extra volumes to mount to Vault servers. This is useful for bringing in extra data that can be referenced by other configurations at a well known path, such as TLS certificates. The value of this should be a list of objects. Each object supports the following keys:
 
       - `type` (`string: required`) -
       Type of the volume, must be one of "configMap" or "secret". Case sensitive.
 
       - `name` (`string: required`) -
       Name of the configMap or secret to be mounted. This also controls the path
-      that it is mounted to. The volume will be mounted to `/vault/userconfig/<name>`.
+      that it is mounted to. The volume will be mounted to `/vault/userconfig/<name>` by default
+      unless `path` is configured.
+      
+      - `path` (`string: /vault/userconfigs`) -
+      Name of the path where a configMap or secret are mounted.  If none specified 
```

Comment:
> configMap or secret are mounted

configMap or secret _is_ mounted (?)

Reply:
Also `If none specified` -> `If not specified`

Reply:
Fixed 👍